client/client_test.go

```diff
@@ -2968,8 +2968,10 @@ func testStargzLazyRegistryCacheImportExport(t *testing.T, sb integration.Sandbo
 			{
 				Type: ExporterImage,
 				Attrs: map[string]string{
-					"name": target,
-					"push": "true",
+					"name":                   target,
+					"push":                   "true",
+					"store":                  "true",
+					"store-allow-incomplete": "true",
```

Comment:
I don't think we need "store-allow-incomplete" as it creates broken images that can't be pushed or extracted. The user has the option to store with blobs or not store at all.

If it is only for tests (and there is no other way), then prefix with `unsafe-internal-`

Reply:
Added `unsafe-internal-` prefix.